__tests__/app.test.js

```diff
@@ -26,6 +26,64 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api", () => {
+  it("GET - 200: Should return a JSON describing all the available endpoints.", () => {
```

Comment:
Very, very expansive. I just expected you to readFile in the test and make sure you were receiving the same information.